### PR TITLE
Add support for array values in useFieldArray methods

### PR DESCRIPTION
**Pull request by andreivt**
This PR is supposed to allow appending, prepending and inserting more than one value at a time since invoking these methods multiple times in a row does not produce the expected result.

### CODE REVIEW

File: src/useFieldArray.ts [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
 } from './types';
+import { useFormContext } from './useFormContext';
+import get from './utils/get';
+import insertAt from './utils/insert';
+import isArray from './utils/isArray';
+import isUndefined from './utils/isUndefined';
+import moveArrayAt from './utils/move';
+import prependAt from './utils/prepend';
+import removeArrayAt from './utils/remove';
+import swapArrayAt from './utils/swap';
```
[the remark this answers is not in the document]
Comment (terandrei) [reply src/useFieldArray.ts]: Sure. Those changed because of organize-imports.

File: src/utils/insert.ts [hunk 2]
```diff
@@ -1,5 +1,5 @@
 export default (data: any, index: number, value?: any) => [
   ...data.slice(0, index),
-  value || null,
+  ...(Array.isArray(value) ? value : [value || null]),
```
Comment (bluebill1049) [inline src/utils/insert.ts]: sorry find one more than, please use `isArray` method

File: src/utils/prepend.ts [hunk 3]
```diff
@@ -1 +1,4 @@
-export default (data: any, value?: any) => [value || null, ...data];
+export default (data: any, value?: any) => [
+  ...(Array.isArray(value) ? value : [value || null]),
```
Comment (bluebill1049) [inline src/utils/prepend.ts]: sorry find one more than, please use `isArray` method

File: src/useFieldArray.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (bluebill1049) [reply src/useFieldArray.ts]: thanks for fixing :)

[the remark this answers is not in the document]
Comment (bluebill1049) [reply src/useFieldArray.ts]: by the way what's this organize-imports? maybe i can use it to apply the whole repo.

[the remark this answers is not in the document]
Comment (terandrei) [reply src/useFieldArray.ts]: https://marketplace.visualstudio.com/items?itemName=alfnielsen.vsc-organize-imports
We're using it at work to keep imports clean.

**Pull request merged**